includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -538,7 +538,10 @@ private function get_used_class_names() {
 		// Find all [class] attributes and capture the contents of any single- or double-quoted strings.
 		foreach ( $this->dom->xpath->query( '//*/@' . Document::AMP_BIND_DATA_ATTR_PREFIX . 'class' ) as $bound_class_attribute ) {
 			if ( preg_match_all( '/([\'"])([^\1]*?)\1/', $bound_class_attribute->nodeValue, $matches ) ) {
-				$classes = array_merge( $classes, $matches[2] );
+				$classes = array_merge(
+					$classes,
+					preg_split( '/\s+/', trim( implode( ' ', $matches[2] ) ) )
```

Comment:
Perhaps better to use `PREG_SPLIT_NO_EMPTY` here than `trim()`?

```suggestion
					preg_split( '/\s+/', implode( ' ', $matches[2] ), PREG_SPLIT_NO_EMPTY )
```

Reply:
Maybe doesn't matter anyway since below we are filtering out values anyway:

https://github.com/ampproject/amp-wp/blob/ee552563eac7de55d08f19ddff0c82ee70b785a2/includes/sanitizers/class-amp-style-sanitizer.php#L550